src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rental = 40;
+  const longTerm = 7;
+  const shortTerm = 3;
+  const totalPrise = days * rental;
+
+  if (days < shortTerm) {
+    return totalPrise;
+  } else if (days >= shortTerm && days < longTerm) {
+    return totalPrise - 20;
+  }
+
+  return totalPrise - 50;
```

Comment:
Avoid using literals in expressions. Define a constant for the discount value to make the code more readable and maintainable, as per the checklist.